great_expectations/expectations/core/expect_column_values_to_be_of_type.py

```diff
@@ -322,7 +322,7 @@ def get_validation_dependencies(
     ):
         # this calls TableExpectation.get_validation_dependencies to set baseline dependencies
         # for the aggregate version of the expectation
-        dependencies = super(ColumnMapExpectation, self).get_validation_dependencies(
+        dependencies = super().get_validation_dependencies(
```

Comment:
We need to keep this as `super(ColumnMapExpectation, self)` - see above.

Reply:
Thanks, @roblim -- again, this is very useful, indeed!  Will be fixed in the next commit.

Reply:
@roblim It turned out that the `view.py` also required `super()` to specify the class.